custom-words.txt

```diff
@@ -2142,6 +2142,7 @@ tilesetconfiguration
 tilesets
 timeasc
 timedesc
+timedout
```

Comment:
Don't add this - timedout is not a single word. Its two words 'timed out'.

Reply:
We can avoid adding this if we fix capitalization elsewhere

Reply:
Looks like that would be breaking, so instead you should remove it here and suppress it with  cSpell.json

Reply:
Removed "timedout" from custom-words.txt and added a suppression in cSpell.json instead